tests/zeroconfig/windows/testdata/Dockerfile.iis.server

```diff
@@ -24,7 +24,8 @@ ENV ACCESS_TOKEN_TMP=$access_token
 RUN `
     $token = $Env:ACCESS_TOKEN_TMP; `
     $collector_msi_path = (dir "c:\setup\splunk-otel-collector-*.msi").FullName; `
-    c:\setup\install.ps1 -access_token $token -msi_path $collector_msi_path -with_dotnet_instrumentation $true -deployment_env zc-iis-test -ingest_url "http://splunk-otel-collector:9943" -trace_url "http://splunk-otel-collector:7276/v2/trace"
+    [string]$network_interface = "127.0.0.1",
```

Comment:
Let's remove this if it's not being used.  Otherwise, it's invalid syntax.
```suggestion
```

Reply:
hmm bad copy/paste? Geez looking.